diff --git a/dist/_chunks/node.mjs b/dist/_chunks/node.mjs
index 1d42edbdef094b20b4d653ec5d069546a12491b5..323cc5eabc5f196b9d47946855a6c06cc298459e 100644
--- a/dist/_chunks/node.mjs
+++ b/dist/_chunks/node.mjs
@@ -13,7 +13,7 @@ function fromNodeUpgradeHandler(handler) {
 }
 const HEARTBEAT_PING = Buffer.from("crossws-ping");
 const nodeAdapter = (options = {}) => {
-	if ("Deno" in globalThis || "Bun" in globalThis) throw new Error("[crossws] Using Node.js adapter in an incompatible environment.");
+	if ("Deno" in globalThis) throw new Error("[crossws] Using Node.js adapter in an incompatible environment.");
 	const hooks = new AdapterHookable(options);
 	const globalPeers = /* @__PURE__ */ new Map();
 	const baseUtils = adapterUtils(globalPeers, options);
